optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/list/externalized/TestdataListSolutionExternalized.java

```diff
@@ -43,9 +43,9 @@ public static TestdataListSolutionExternalized generateUninitializedSolution(int
         return solution;
     }
 
-    private List<TestdataListValueExternalized> valueList;
-    private List<TestdataListEntityExternalized> entityList;
-    private SimpleScore score;
+    public List<TestdataListValueExternalized> valueList;
+    public List<TestdataListEntityExternalized> entityList;
+    public SimpleScore score;
```

Comment:
Made public so the unit test will also run with the GIZMO SolutionCloner

Reply:
Look like I forgot I the GizmoSolutionCloner test does not follow the normal Gizmo route; it get passed a `memoizedSolutionOrEntityDescriptorMap` which CAN contains getters and setters, unlike user application code. If it was in user code, the public field restrictions apply

Reply:
That is strange because if I turn on Gizmo access in optaweb-vehicle-routing using `quarkus.optaplanner.solver.domain-access-type=GIZMO` I can't see anything failing even though it uses private fields and public getters/setters in the domain model.

Reply:
What I am talking about is for plain Java/Spring Boot; Quarkus is special since is generates getter methods that are guaranteed to be simple.

Reply:
Also, GIZMO is the default in Quarkus